### PR TITLE
[sig-windows] Move remaining Windows jobs to community cluster

### PR DESCRIPTION
**Pull request by jsturtevant**
Move the remaining windows jobs running on Capz release branches to the community clusters

/sig windows
/assign @nawazkh @ritikaguptams

### CODE REVIEW

**Review by willie-yao — APPROVED** [review]
/lgtm

File: config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml [hunk 1]
```diff
@@ -256,21 +269,19 @@ periodics:
       - "perfDashPrefix: soak-tests-capz-windows-2019"
       - "perfDashJobType: windows"
     labels:
-      preset-service-account: "true"
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-      preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true" # Sets REGISTRY which is needed when building CCM/CNM images
```
Comment (nawazkh) [inline config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml:274]: Should we be dropping `reset-azure-anonymous-pull: "true"` as well?

**Pull request merged**